<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormScene</class>
 <widget class="QWidget" name="FormScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>532</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollAreaScene">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaSceneLayout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>398</width>
        <height>530</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="mc::gui::TreeScene" name="treeScene">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">Components</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonAdd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboTypes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mc::gui::TreeScene</class>
   <extends>QTreeWidget</extends>
   <header>gui/TreeScene.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
